### ISSUE TITLE
Dynamic port when running via .jar

### ISSUE THREAD

**Issue by nicoschoenmaker** [tomakehurst/wiremock#898]
TL;DR; Would be awesome if there was an ability for selecting a dynamic port when running the standalone jar.

### Why?
- It's required for running concurrent tests.
- It's possible when running the server [via Java](http://wiremock.org/docs/getting-started/#random-port-numbers)

### How?
Possibly with a `--dynamic-port` option, similar to `--port`.
There would need to be a way to feed the chosen port back to the process running the jar file.

### Workaround
Workaround might be to keep trying to start the jar with new ports till it doesn't exit with a non-zero exit code after x time. Doesn't sound very stable to me :stuck_out_tongue_closed_eyes: 

Alternate ideas very much welcome, only started researching wiremock today!

**Comment by tomakehurst** [comment]
It's already possible to randomly assign a port by passing `--port 0`.

However, it doesn't print the port it's bound to, so at the moment there's a bit of work to find it out from the OS.

Feel like creating a PR that prints the actual port at startup?

**Comment by nicoschoenmaker** [comment]
@tomakehurst yeah, opened a PR for it. Do you have time to provide feedback on #900?
